10-post-form/src/app.js

```diff
@@ -1,14 +1,11 @@
 const express = require('express');
 const path = require('path');
 const favicon = require('serve-favicon');
-// import 'body-parser' module
```

Comment:
We still need to import the module. See my comment below to understand why.

Reply:
oooops ! sorry I will do it :+1: